src/item.c

```diff
@@ -163,7 +163,7 @@ int is_obstacle(int w) {
     }
     switch (w) {
         case EMPTY:
-        case CLOUD:
```

Comment:
If the CLOUD case isn't necessary anymore, I'd just delete it instead of commenting it out. Or if you're commenting it out, add some explanation.

Reply:
I was planning on commenting it out because it resolves US 2.2.2 Make Fog Not Transparent (which was much more trivial than I thought it was when assigned).